draft-ietf-ice-rfc5245bis.xml

```diff
@@ -1200,14 +1200,11 @@ address, this value SHOULD be set to 65535. More generally, if there
 are multiple candidates for a particular component for a particular
 media stream that have the same type, the local preference MUST be
 unique for each one. In this specification, this only happens for
-multihomed hosts or if an agent is using multiple TURN servers. If a
+multihomed hosts or if an agent is using multiple TURN servers.  If a
 host is multihomed because it is dual-stack, the local preference
-SHOULD be set equal to the precedence value for IP addresses described
-in RFC 6724 <xref target="RFC6724"/>. If the host operating system
-provides an API for discovering preference among different addresses,
-those preferences SHOULD be used for the local preference to
-prioritize addresses indicated as preferred by the operating system.
-</t>
+should be set according to the current best practice described in <xref
+target="I-D.ietf-ice-dualstack-fairness"> ICE Multihomed and IPv4/IPv6
+Dual Stack Fairness </xref>.</t>
```

Comment:
nit: s/ICE Multihomed and IPv4/IPv6 Dual Stack Fairness/"ICE Multihomed and IPv4/IPv6 Dual Stack Fairness" document/
Or drop the draft name; usually they are not mentioned in this kind of context.